### ISSUE TITLE
Version in "./docker-compose_base.yml" is unsupported.

### ISSUE THREAD

**Issue by aaronott** [DefectDojo/django-DefectDojo#1126]
**Bug description**
This is a duplicate of https://github.com/DefectDojo/django-DefectDojo/issues/1032 but on the latest version of defectdojo.

I'm on a fresh install of Ubuntu 18.04.2 with nothing else installed. I'm following the Getting Started guide at https://defectdojo.readthedocs.io/en/latest/getting-started.html#docker-compose-install and when I do a `docker-compose up` I receive the following error:

> ERROR: Version in "./docker-compose_base.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

Following the "Solution" in that post (updating 3.7 to 3.3) I was able to successfully perform the `docker-compose up` and get DefectDojo running.

**Steps to reproduce**
1. Fresh install Ubuntu 18.04.2
2. Follow steps from Getting started guide: https://defectdojo.readthedocs.io/en/latest/getting-started.html#docker-compose-install
3. See error

**Steps to bypass error**
1. Update `./docker-compose_base.yml` set version from 3.7 to 3.3
2. Update `./docker-compose_uwsgi-release.yml` set version from 3.7 to 3.3
3. Add `docker` group to current user `sudo usermod -aG docker ${USER}`
4. Log out and back in.
5. Switch back to django-DefectDojo directory
6. `. docker/aliases_release.sh`
7. `docker-compose up`

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [x] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
aott@defect:/home/aott/django-DefectDojo$ docker --version
`Docker version 18.09.2, build 6247962`
aott@defect:/home/aott/django-DefectDojo$ docker-compose --version
`docker-compose version 1.22.0, build unknown`
aott@defect:/home/aott/django-DefectDojo$ cat /etc/issue
`Ubuntu 18.04.2 LTS \n \l`
aott@defect:~/django-DefectDojo$ git show -s --format="%h: %s [%ci]"
`2a8e2ef5: Merge pull request #1109 from wurstbrot/patch-1 [2019-05-06 12:13:47 -0400]`

**Comment by dr3dd589** [comment]
run this 
```
docker-compose -f docker-compose_base.yml -f docker-compose_uwsgi-release.yml up
```

https://github.com/DefectDojo/django-DefectDojo/blob/master/DOCKER.md

**Comment by ppiotr3k** [comment]
Hi @aaronott, thank you for your interest in DefectDojo and for taking the time for submitting an issue.

> I'm following the Getting Started guide at https://defectdojo.readthedocs.io/en/latest/getting-started.html#docker-compose-install

In the pointed section, the first 2 lines are:
```
    Latest version of Docker
    Latest version Docker Compose
```
The versions available in Ubuntu repositories for 18.04 LTS are too old.
As indicated in the documentation, please use **latest** versions, at least: 

 :heavy_check_mark: `docker` 18.09.4
:heavy_multiplication_x:  `docker-compose` 1.24.0

Therefore, by following the [Docker official install documentation for Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/) you should be able to resolve your problem.
